DoublyLinkedList/.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
this can also be ignored and deleted. 

only include files in repo that are needed by other collaborators and are essential to the project.

Reply:
yes sir now I understand